### PR TITLE
Prototype downloading pixel board into .png

### PR DESCRIPTION
**Pull request by vgrichina**

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const { connect, Account, keyStores: { InMemoryKeyStore } } = require('near-api-js');
+
+(async function () {
+    const config = require('./config')(process.env.NODE_ENV || 'development')
+    // TODO: Why no default keyStore?
+    const keyStore = new InMemoryKeyStore();
+    const near = await connect({...config, keyStore});
+    const account = await near.account('berryclub.ek.near');
+    const pixelsState = await account.viewState('p');
```
Comment (vgrichina) [inline backend/app.js]: @evgenykuzyakov is there any way to call https://docs.near.org/docs/api/rpc#view-contract-state with specific block index?

Comment (vgrichina) [reply backend/app.js]: This code uses https://github.com/near/near-api-js/pull/452

Comment (bowenwang1996) [reply backend/app.js]: yeah you can pass `block_id` in the query

Comment (evgenykuzyakov) [reply backend/app.js]: In the RPC params, change `"finality": "..."` to `"block_id": <BLOCK_HEIGHT>`

**Comment by vgrichina** [comment]
Moved into it's own repo:
https://github.com/vgrichina/berry-wayback

**Pull request closed without merging**